arm9/src/unk_0202E29C.c

```diff
@@ -917,63 +913,49 @@ THUMB_FUNC struct UnkStruct_0202E29C_sub4 *FUN_0202EF44(s32 param0)
 
 THUMB_FUNC struct PlayerData *FUN_0202EF70()
 {
-    return UNK_021C59E8.unk8->unkCB8;
+    return UNK_021C59F0->unkCB8;
 }
 
 THUMB_FUNC struct PlayerData *FUN_0202EF84(int param0)
 {
-    if (UNK_021C59E8.unk8->unkC84[param0] == 0)
+    if (UNK_021C59F0->unkC84[param0] == 0)
     {
         return NULL;
     }
 
-    struct UnkStruct_0202E29C_sub4 *r0 = &UNK_021C59E8.unk8->unk054[param0].unk50;
+    struct UnkStruct_0202E29C_sub4 *r0 = &UNK_021C59F0->unk054[param0].unk50;
 
     return &r0->unk10;
 }
 
 THUMB_FUNC void FUN_0202EFAC(u8 *param0, s32 param1)
 {
-    if (UNK_021C59E8.unk8 != NULL)
+    if (UNK_021C59F0 != NULL)
     {
         GF_ASSERT(param1 < 8);
 
-        MI_CpuCopy8(param0, UNK_021C59E8.unk8->unkC54[param1], 6);
+        MI_CpuCopy8(param0, UNK_021C59F0->unkC54[param1], 6);
     }
 }
 
 #ifdef NONMATCHING
 THUMB_FUNC BOOL FUN_0202EFE4(u8 *param0)
```

Comment:
I assume that this func still does not match?

Reply:
Aye. But it's closer.